app/views/ppm_estimator/_estimate_table.html.erb

```diff
@@ -1,11 +1,90 @@
-<div class="ppm-estimate-table usa-grid-full">
-  <div class="usa-width-one-half">
-    <span class="ppm-estimate-detail-total">Your PPM Incentive Estimate:</span>
-    <span id="incentive-range" class="ppm-estimate-amount-total">$<%= @ppm_estimator.total_incentive_range.min %>–$<%= @ppm_estimator.total_incentive_range.max %></span>
+<span class="ppm-estimate-header">Your PPM Incentive Estimate:</span>
+<a href="javascript:window.print()" class="ppm-estimate-print">Print</a>
+<div class="ppm-estimate-table">
+  <div class="ppm-estimate-locations">
+    From <%= @ppm_estimator.start_zipcode_info.city %>, <%= @ppm_estimator.start_zipcode_info.state.abbr %> <%= inputs[:start] %> to <%= @ppm_estimator.end_zipcode_info.city %>, <%= @ppm_estimator.end_zipcode_info.state.abbr %> <%= inputs[:end] %><br>
+    <%= tag.div class: 'ppm-estimator-map', id: 'ppm-estimator-map',
+      data: {
+        'icon-url': asset_path('leaflet/dist/images/marker-icon.png'),
+        'icon-retina-url': asset_path('leaflet/dist/images/marker-icon-2x.png'),
+        'shadow-url': asset_path('leaflet/dist/images/marker-shadow.png'),
+        'start-latitude': @ppm_estimator.start_zipcode_info.lat.to_s,
+        'start-longitude': @ppm_estimator.start_zipcode_info.lon.to_s,
+        'end-latitude': @ppm_estimator.end_zipcode_info.lat.to_s,
+        'end-longitude': @ppm_estimator.end_zipcode_info.lon.to_s
+      }
+    %>
   </div>
-  <div class="usa-width-one-half">
-    <span class="ppm-estimate-detail-total">Maximum Advance Payment:</span>
-    <span id="advance-range" class="ppm-estimate-amount-total">$<%= @ppm_estimator.advance_range.min %>–$<%= @ppm_estimator.advance_range.max %> (<%= @ppm_estimator.advance_percentage %>%)</span>
+
+  <div class="ppm-estimate-inputs">
+    <div class="ppm-estimate-table-cell">
+      <%- case inputs[:branch] %>
```

Comment:
This switch is verbose enough that it seems to violate either the principle of DRY or the principle of keeping logic out of the view. Any suggestions?